src/main/java/org/broadinstitute/consent/http/db/SamDefaults.java

```diff
@@ -0,0 +1,81 @@
+package org.broadinstitute.consent.http.db;
+
+import com.google.api.client.http.HttpStatusCodes;
+import java.io.InputStream;
+import java.nio.charset.Charset;
+import java.util.Date;
+import org.apache.commons.io.IOUtils;
+import org.broadinstitute.consent.http.models.AuthUser;
+import org.broadinstitute.consent.http.models.sam.EmailResponse;
+import org.broadinstitute.consent.http.models.sam.TosResponse;
+import org.broadinstitute.consent.http.models.sam.UserStatus;
+import org.broadinstitute.consent.http.models.sam.UserStatusDiagnostics;
+import org.broadinstitute.consent.http.models.sam.UserStatusInfo;
+
+/**
+ * This class represents default/mock objects from Sam that can be used when Sam is unavailable
+ */
+public class SamDefaults {
+
+  public UserStatusInfo getDefaultUserStatusInfo(AuthUser authUser) {
+    UserStatusInfo userStatusInfo = new UserStatusInfo();
+    userStatusInfo.setAdminEnabled(false);
+    userStatusInfo.setEnabled(true);
+    userStatusInfo.setUserEmail(authUser.getEmail());
+    userStatusInfo.setUserSubjectId(authUser.getEmail());
+    return userStatusInfo;
+  }
+
+  public UserStatusDiagnostics getDefaultUserStatusDiagnostics() {
+    UserStatusDiagnostics userStatusDiagnostics = new UserStatusDiagnostics();
+    userStatusDiagnostics.setAdminEnabled(false);
+    userStatusDiagnostics.setEnabled(true);
+    userStatusDiagnostics.setTosAccepted(true);
```

Comment:
This suggests that a new user who hasn't accepted the TOS will have the diagnostics report back that they have accepted it. Is this expected?

Reply:
Yes, this is intentional. Without ToS, the user cannot proceed in DUOS. If Sam is down, there's no way to determine this state without replicating it on our side - which I am not in favor of doing. When Sam comes back, the user will eventually need to accept ToS.